### PR TITLE
Enable quotes validation

### PR DESCRIPTION
**Pull request by HunseopJeong**
- Add the 'quotes' option to enable quotes validation
- Add the 'avoidEscape' option to allow some exceptions
  (https://eslint.org/docs/rules/quotes#avoidescape)
- Fix wrong quotes

Signed-off-by: Hunseop Jeong <hs85.jeong@samsung.com>

### CODE REVIEW

File: libs/brackets-server/.eslintrc [hunk 1]
```diff
@@ -50,6 +50,7 @@
 
         "no-trailing-spaces": 0,
         "eol-last": 0,
+        "quotes": ["error", "double", { "avoidEscape": true }]
```
Comment (Kwanghyuk-Kim) [inline libs/brackets-server/.eslintrc]: What about other brackets code ?
Doesn't this be necessary for embedded-ext only ?

Comment (HunseopJeong) [reply libs/brackets-server/.eslintrc]: We apply this rule to the files following directories :
```
'libs/brackets-server/embedded-ext/**/*.js',
'libs/brackets-server/hacks/**/*.js',
'libs/brackets-server/lib/**/*.js',
```

File: public/js/project.js [hunk 2]
```diff
@@ -87,7 +87,7 @@ $('#new-project').on('show.bs.modal', function() {
     webBtn.addClass('btn-primary');
   });
 
-  $("#empty-checkbox").on('change', function() {
+  $('#empty-checkbox').on('change', function() {
```
Comment (Kwanghyuk-Kim) [inline public/js/project.js]: What abou making this as another patch ?
Modification of this file doesn't seem to match with title of patch although you explained it on sub comment.

Comment (HunseopJeong) [reply public/js/project.js]: I hope to fix it on this PR :(
I comment it on description.

Comment (Kwanghyuk-Kim) [reply public/js/project.js]: for now, I will agree to it. but from next time, let's apply one change per a patch.

**Pull request merged**